.github/workflows/CI.yml

```
@@ -22,3 +22,5 @@ jobs:
       - run: cargo fmt -- --check
       - run: cargo test
       - run: cargo clippy
+      # Ensure that the `Cargo.lock` file was correctly updated.
```

This checks any modified file, right?
```suggestion
      # Ensure that no untracked or tracked files have been added or modified.
```

---

You wanna hide the actual reason hehehe. :D

Updating then. :wink: